src/pageObject.js

```diff
@@ -1,16 +1,42 @@
-'use strict';
+"use strict";
 
 class Header {
+  get logo() {
+    return "Conduit logo";
+  }
+}
 
-};
-
-class PageObject {
+class PageObject extends Header {
+  constructor(url) {
+    super();
+    this.url = url;
+    this.header = new Header();
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
```

Comment:
The `ArticlePage` class extends the `PageObject` class, which is a good use of inheritance since an `ArticlePage` is a type of `PageObject`. However, make sure that the `ArticlePage` class only adds or overrides methods and properties that are specific to an `ArticlePage`.